GOVERNANCE.md

```diff
@@ -3,16 +3,31 @@
 This document describes the governance process under which the Serverless
 Working Group (WG) will manage this repository.
 
-## Working Group Meetings
+## Working Group Process
 
 In order to provide equitable rights to all Working Group members,
 the following process will be followed:
 
 * Official WG meetings will be announced at least a week in advance.
-* Proposed changes to any document will be done via a Pull Request (PR).
-* PRs will be reviewed during official WG meetings.
-* During meetings, priority will be given to PRs that appear to be ready for
-  a vote over those that appear to require discussions.
+* Proposed changes to any document will be done via a Pull Request (PR)
+* PRs will be reviewed in advance of WG meetings
+* Active members will vote on PRs in advance of Working Group meetings to
+  promote them to active consideration
+* Changes to the [spec.md](spec.md) require active member participation:
+  * if there are no negative votes, then any PR with 3 affirmative votes can be
+    raised for consideration
+  * if there are negative votes, then 5 affirmative votes and a code sample
+  illustrating the benefit of the change or drawback of current spec which would
```

Comment:
I'm not sure why we should force a higher bar (ie. code) for controversial PRs than non-controversial ones. My feeling is that seeing running code won't actually change anyone's mind since anyone can code anything. I think people should discuss usecases and requirements to explain why a change is needed - but I think that should be true for all PRs.

Reply:
This is an attempt to move us toward working code samples that illustrate a point, rather than taking action based on the loudest voices